### PR TITLE
Add more sceKernelGetGameInfo 's nids

### PR DESCRIPTION
**Pull request by sum2012**
https://github.com/uofw/uofwinst/blob/master/SystemControl/nid_660_data.c#L81

### CODE REVIEW

**Review by gid15 — COMMENTED** [review]
Doing a small research, I have found that the NID of this method has changed with the PSP versions (I have only checked the below PSP versions):
2.60: 0xCD617A94
5.00: 0x0D547E7F
6.60: 0xEF29061C

**Comment by sum2012** [comment]
How about  0xAB5E85E5 ?
https://github.com/uofw/uofwinst/blob/master/SystemControl/nid_660_data.c#L82

**Comment by gid15** [comment]
6.20: 0xAB5E85E5
6.35: 0x3C4C5630

**Comment by sum2012** [comment]
There is no version 1.5 of sceKernelGetGameInfo ?

File: src/jpcsp/HLE/modules/SysMemForKernel.java [hunk 1]
```diff
@@ -544,7 +544,10 @@ public int sceKernelSetDNAS(int dnas) {
     }
 
     @HLEUnimplemented
-    @HLEFunction(nid = 0xEF29061C, version = 150)
+	@HLEFunction(nid = 0xCD617A94, version = 260)
+	@HLEFunction(nid = 0x0D547E7F, version = 500)
+	@HLEFunction(nid = 0xEF29061C, version = 635)
```
Comment (gid15) [inline src/jpcsp/HLE/modules/SysMemForKernel.java:549]: 6.35: 0x3C4C5630
6.60: 0xEF29061C

**Review by gid15 — COMMENTED** [review]
I could not find the method in 1.50...

**Pull request merged**